mantidproject/mantid
# QLRun Mantid Algorithm

Fixes #13826

QLRun is now a Mantid algorithm as opposed to the previously used script. This is normally run in the Indirect Bayes : Quasi interface.
# To Test
- Open Quasi (Interfaces > Indirect > Bayes > Quasi)
- Load in a sample and resolution
  - Sample = `ExternalData\Testing\Data\DocTest\irs26176_graphite002_red.nxs`
  - Resolution = `ExternalData\Testing\Data\DocTest\irs26173_graphite002_red.nxs`
- All the default options should be fine in the interface
- Click Run 
  - _There is currently no Progress Tracking on this algorithm. This will be in #13944 after this branch is merged._
- Ensure the following workspaces are produced:
  - `irs26176_graphite002_QLd_Prob` <- MatrixWorkspace
  - `irs26176_graphite002_QLd_Result` <- MatrixWorkspace
  - `irs26176_graphite002_QLd_Workspace` <- WorkspaceGroup
    - The WorkspaceGroup should contain workspaces 0-9
- Ensure the mini plot is updated in the interface and the result looks reasonable.
  _If you wish to ensure the clarity of the data, you could potentially compare it to data produced by 3.5 version of Mantid. No changes have actually been made to the workflow of the algorithm._
